Uninstall/Pre-Uninstall/Uninstall-Software/Uninstall-Software.ps1

```diff
@@ -220,10 +220,13 @@ function Get-InstalledSoftware {
 
             }
             'HKCU' {
-                [string]::Format('registry::HKEY_CURRENT_USER\{0}', $PathFragment)
+                # There is no Wow6432Node Uninstall key for HKEY_CURRENT_USER
+                [string]::Format('registry::HKEY_CURRENT_USER\{0}', $PathFragment.Replace('\Wow6432Node',''))
```

Comment:
I have never seen anything under there except Active Setup entries though.

It's not like Wow6432Node under HKLM, where a 32-bit process gets secretly redirected to HKLM\Software\Wow6432Node when looking at HKLM\Software.

Won't do any harm having it in there though, so will revert that bit just in case.